pages/en/lb3/Defining-middleware.md

```diff
@@ -66,6 +66,8 @@ For example, for the "initial" phase, middleware executes in this order:
 Middleware within a single subphase executes in the order in which it is registered. However, you should not rely on such order.
 Always explicitly order the middleware using appropriate phases when order matters.
 
+**Note**: If a route is found in the `route` phase, no middlewares after that will be called within loopback, including `route:after`. This is due to the express behavior.
```

Comment:
I find the proposed text possibly misleading.

- This behavior is not specific for `route` phase, it applies to all middleware independently on where it is registered. For example, `cors` middleware can handle `OPTIONS` request and terminate the middleware chain early on.
- The behavior kicks in when a middleware or a route does not call `next()`. 
- Routes created for LoopBack remote methods never call `next()`.
- Express routes serving content typically don't call `next` either.
- By default, strong-remoting will "own" the URL namespace (typically all paths starting with `/api`). When there is no remote method matching the requested path + verb, then 404 error is triggered. This behavior can be disabled via `rest.handleUnknownPaths` in `server/config.json`, see [Remoting properties](https://loopback.io/doc/en/lb3/config.json.html#remoting-properties)